docs/usage.md

```diff
@@ -83,29 +226,293 @@ nextflow pull nf-core/oncoanalyser
+
+```csv title="samplesheet.csv"
+group_id,subject_id,sample_id,sample_type,sequence_type,filetype,filepath
+P1_wgts,P1,SA,normal,dna,bam,/path/to/P1.SA.normal.dna.wgs.bam
+P1_wgts,P1,SB,tumor,dna,bam,/path/to/P1.SB.tumor.dna.wgs.bam
+P1_wgts,P1,SC,tumor,rna,bam,/path/to/P1.SC.tumor.rna.wts.bam
+```
+
+```bash
+nextflow run nf-core/oncoanalyser \
+  -profile docker \
+  -revision 1.0.0 \
+  --mode wgts \
+  --genome GRCh38_hmf \
+  --prepare_reference_only \
+  --input samplesheet.csv \
+  --outdir prepare_reference/
+```
+
+Executing the above command will download and unpack default reference data without running any analysis, and once
+complete the prepared reference files can found in `./prepare_reference/reference_data/1.0.0/<datetimestamp>/`. It is
+recommended to remove the Nextflow work directory after staging data to free disk space.
+
+For `oncoanalyser` to use locally staged reference data a custom config can be used:
+
+```groovy title="refdata.local.config"
+params {
+
+    genomes {
+        GRCh38_hmf {
+            fasta           = "/path/to/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna"
+            fai             = "/path/to/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.fai"
+            dict            = "/path/to/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.dict"
+            bwamem2_index   = "/path/to/bwa-mem2_index/"
+            gridss_index    = "/path/to/gridss_index/"
+            star_index      = "/path/to/star_index/"
+        }
+    }
+
+    ref_data_hmf_data_path        = "/path/to/hmftools_data/"
+    ref_data_panel_data_path      = "/path/to/tso500_panel_data/"
+    ref_data_virusbreakenddb_path = "/path/to/virusbreakenddb/"
+}
+```
+
+Specific reference files can also be downloaded directly from the hosting service with the corresponding URL.
+
+##### Reference data URLs
+
+_GRCh37 genome (Hartwig) [`GRCh37_hmf`]_
+
+| Type                 | Name                                                                                                                                                                                         |
+| -------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| FASTA                | [Homo_sapiens.GRCh37.GATK.illumina.fasta](https://pub-cf6ba01919994c3cbd354659947f74d8.r2.dev/genomes/GRCh37_hmf/24.0/Homo_sapiens.GRCh37.GATK.illumina.fasta)                               |
+| FASTA index          | [Homo_sapiens.GRCh37.GATK.illumina.fasta.fai](https://pub-cf6ba01919994c3cbd354659947f74d8.r2.dev/genomes/GRCh37_hmf/24.0/samtools_index/1.16/Homo_sapiens.GRCh37.GATK.illumina.fasta.fai)   |
+| FASTA seq dictionary | [Homo_sapiens.GRCh37.GATK.illumina.fasta.dict](https://pub-cf6ba01919994c3cbd354659947f74d8.r2.dev/genomes/GRCh37_hmf/24.0/samtools_index/1.16/Homo_sapiens.GRCh37.GATK.illumina.fasta.dict) |
```

Comment:
can the GATK reference genomes already available in igenomes be used here?

Reply:
That's a good question - I've compared the available GATK hg19 and GRCh37 genomes from iGenomes to the above Hartwig reference but there appears to be some differences (FASTQ record description lines and all newline breaks were removed for checksum):

```text
Source          Build  Filename                                 MD5_checksum
Hartwig         GRCh37 Homo_sapiens.GRCh37.GATK.illumina.fasta  92390ef930f6c0dd697f8f587a11e154
iGenomes [GATK] GRCh37 human_g1k_v37_decoy.fasta                0e66f50435336d630fa69d52df51c2be
iGenomes [GATK] hg19   ucsc.hg19.fasta                          be13c27319d550e09a6d50772f28b3c1
```

While this isn't a comprehensive comparison, it's enough that I'd prefer to not switch away from the recommended Hartwig genome even if there are naming similarities and some shared/partial provenance.